File: C#/hello.cs
//$ sudo apt update[ENTER]
//$ sudo apt install mono-complete[ENTER]	...Once
//$ mcs hello.cs[ENTER]
//$ mono hello.exe[ENTER]
//$ rm hello.exe[ENTER]
using System;
namespace HelloWorld{
	class hello{
		static void Main(string[] args){
			Console.WriteLine("Hello, world!");
			//Console.ReadLine();
		}
	}
}

File: Objective-C/hello.m
//$ sudo apt-get install gobjc[ENTER]...Once
//$ gcc -o hello hello.m[ENTER]
//$ ./hello[ENTER]
#import <Foundation/Foundation.h>
int main (int argc, const char * argv[]){
	NSAutoreleasePool * pool = [[NSAutoreleasePool alloc] init];
	NSLog (@"hello world");
	[pool drain];
	return 0;
}
